Forwarded: https://github.com/Rich-Harris/magic-string/pull/185

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,10 +1,10 @@
 import buble from 'rollup-plugin-buble';
-import resolve from 'rollup-plugin-node-resolve';
+import nodeResolve from '@rollup/plugin-node-resolve';
 import replace from 'rollup-plugin-replace';
 
 const plugins = [
 	buble({ exclude: 'node_modules/**' }),
-	resolve(),
+	nodeResolve(),
 	replace({ DEBUG: false })
 ];
 
